src/themes/default/pages/Category.vue

```diff
@@ -118,7 +126,8 @@ export default {
     return {
       mobileFilters: false,
       defaultColumn: 3,
-      loadingProducts: false
+      loadingProducts: false,
```

Comment:
is this `loadingProducts` used anywhere?

Reply:
yeah in `onBottomScroll` so I kept that